perma_web/perma/templates/user_management/create-link.html

```diff
@@ -153,6 +153,15 @@ <h3 class="link-header body-ah">Your Perma Links</h3>
         host = "{{ request.get_host }}";
   </script>
 
+  {% if org_user %}
+    <script>
+      if (links_remaining == 10){
+        var message = "Your personal links for the month are almost used up! Create more links in 'unlimited' folders."
+        Helpers.informUser(message, 'danger');
+      }
+    </script>
+  {% endif %}
+
```

Comment:
This probably wants to live in create.module.js. Could use a custom JS variable `is_org_user`, if you can't get that status from `current_user` defined just above.

(This is nitpicky now, but will be necessary post-JS-refactor when Helpers stops being in the global namespace and we start reporting JS test coverage and so on.)